### DISCUSSION TITLE
Is there any way to tell DocFX to discard styles/*.map files from its build output?

### DISCUSSION THREAD

**Discussion by dav-tb** [dotnet/docfx#8785] [Q&A]
AFAIK, these files are only useful for client-side debugging scenarios, right? Can I safely remove them after the build, or is there a special "production"-mode build config in `DocFX` I'm unaware of that takes them out?

**Comment by yufeih** [comment]
> Can I safely remove them after the build?
Yes. There is currently no config exposed to docfx to skip the copy of these .map files.
